aspnetcore/razor-pages/sdk.md

```diff
@@ -55,6 +55,8 @@ To use the Razor SDK to build class libraries containing Razor views or Razor Pa
 
 ::: moniker-end
 
+* Package references need to be included in the project containing razor views. If a downstream dependency includes required packages you still need to explicitly reference the preceding dependencies.
```

Comment:
I'm not sure if this statement helps. You specifically need a reference to the `Microsoft.AspNetCore.Mvc` package in Razor Class libraries for 2.x targeting projects since it has build targets which are intended to be non-transitive. Other references can be picked up transitively. I'd also put this near the bottom of the page. Our general guidance is to start from a RCL template which has the reference, so this is a fairly uncommon scenario.